src/index.js

```diff
@@ -53,4 +54,7 @@ if (config.plugAndPlay) {
     handleUnknownDeviceData();
 }
 
+const statisticSavers = {};
```

Comment:
As I proper understand this object ```statisticSavers``` will be used for select data for statistic, right?